src/sentry/api/endpoints/organization_events.py

```diff
@@ -94,28 +93,65 @@
 
 API_TOKEN_REFERRER = Referrer.API_AUTH_TOKEN_EVENTS.value
 
-DEFAULT_RATE_LIMIT = 30
-DEFAULT_RATE_LIMIT_WINDOW = 1
-DEFAULT_CONCURRENT_RATE_LIMIT = 15
-
-DEFAULT_EVENTS_RATE_LIMIT_CONFIG = {
-    "GET": {
-        RateLimitCategory.IP: RateLimit(
-            DEFAULT_RATE_LIMIT, DEFAULT_RATE_LIMIT_WINDOW, DEFAULT_CONCURRENT_RATE_LIMIT
-        ),
-        RateLimitCategory.USER: RateLimit(
-            DEFAULT_RATE_LIMIT, DEFAULT_RATE_LIMIT_WINDOW, DEFAULT_CONCURRENT_RATE_LIMIT
-        ),
-        RateLimitCategory.ORGANIZATION: RateLimit(
-            DEFAULT_RATE_LIMIT, DEFAULT_RATE_LIMIT_WINDOW, DEFAULT_CONCURRENT_RATE_LIMIT
-        ),
-    }
-}
-
+LEGACY_RATE_LIMIT = dict(limit=30, window=1, concurrent_limit=15)
+# reduced limit will be the future default for all organizations not explicitly on increased limit
+DEFAULT_REDUCED_RATE_LIMIT = dict(
+    limit=1000, window=300, concurrent_limit=15  # 1000 requests per 5 minutes
+)
+DEFAULT_INCREASED_RATE_LIMIT = dict(limit=50, window=1, concurrent_limit=50)
+
+
+def rate_limit_events(
```

Comment:
where this is called, have we not grabbed the organization yet? (i.e., is that why we're not passing in the org directly)?

Reply:
@JoshFerge  as far as I understand this is called _before_ the raw request parameters are converted to objects.

This is where it eventually gets called:

https://github.com/getsentry/sentry/blob/92e92bf00f5dc5abe5f48ab96fa61b87226bdb24/src/sentry/ratelimits/utils.py#L169-L170

Which in turn is called in `process_view(...)` of `RateLimitMiddleware`:
https://github.com/getsentry/sentry/blob/92e92bf00f5dc5abe5f48ab96fa61b87226bdb24/src/sentry/middleware/ratelimit.py#L65